### DISCUSSION TITLE
New FRR version v8.5.0

### DISCUSSION THREAD

**Discussion by jbemmel** [ipspace/netlab#779] [General]
New release came out 3 weeks ago: https://github.com/FRRouting/frr/releases

Highlights:
* Support for BGP AddPath
* L3 route-target auto/wildcard configuration
* SRv6 uSID (microSID) implementation and per-VRF SRv6 SID

New Docker image isn't available (yet?), will monitor

**Comment by jbemmel** [comment]
Image has moved to ```quay.io/frrouting/frr:8.5.2``` - I propose we update the default

(all versions: https://quay.io/repository/frrouting/frr?tab=tags&tag=latest)

**Reply by ipspace** [reply]
Sure. Just hope they haven't changed config syntax -- a previous similar update has broken some functionality (can't figure out what it was though)
